modules/appeals_api/config/locales/en.yml

```diff
@@ -1,9 +1,9 @@
 en:
   appeals_api:
       errors:
-        claimant_info: "if any claimant info is present, claimant %{field} must also be present"
-        contact_info_presence: "at least one must be included: '/data/attributes/veteran', '/data/attributes/claimant'"
+#        claimant_info: "if any claimant info is present, claimant %{field} must also be present"
+        contact_info_presence: "at least one must be included: '/data/attributes/veteran'"
```

Comment:
Check whether this translation is still needed. I fit is, the text doesn't make sense anymore. Just indicate the veteran info is required.